rudder-web/src/main/scala/com/normation/rudder/web/components/DirectiveEditForm.scala

```diff
@@ -427,6 +430,59 @@ class DirectiveEditForm(
     }
     s"${version} ${deprecationInfo}"
   }
+  private[this] val globalOverrideText = configService.rudder_policy_overridable().getOrElse("") match {
+    case true  =>
+      <div>
+        You may override the agent policy mode on this directive.
+        If set to <b>Audit</b> this directive will never be enforced.
+        If set to <b>Enforce</b> this directive will appply necessary changes except on Nodes with a <b>Verify</b> override setting.
+      </div>
+    case false =>
+      <p>
+        Currrent global settings do not allow this mode to be overriden on a per-directive bases. You may change this in <b>Settings</b>,
+        or contact your Rudder administrator about this.
+      </p>
+  }
+    private[this] val policyModes = {
+      val globalPolicyModeName = configService.rudder_policy_mode_name().getOrElse("") match {
+        case Verify  => "Audit"
```

Comment:
You should not rename modes to Audit or Enforce, you should do "policyMode.name.capitalize()"